benchmarks/python/test_dropout_layernorm_bwd.py

```diff
@@ -4,7 +4,12 @@
 import pytest
 from nvfuser import FusionDefinition, DataType
 from nvfuser.pytorch_utils import torch_dtype_to_nvfuser_dtype
-from .core import run_benchmark, clear_cuda_cache
+from .core import (
+    run_benchmark,
+    clear_cuda_cache,
+    unary_bwd_torch,
+    compute_total_iobytes,
+)
```

Comment:
You could just do `import core` then qualify the functions at the call sites. Not really necessary, but if the import list gets bigger you could consider switching.